client/js/global.js

```diff
@@ -0,0 +1,64 @@
+require('../css/global.scss');
+const $ = require('jQuery');
+
+/**
+* fm-overlays-global.js
+*
+* @created 1/15/16 5:33 PM
+* @author Alley Interactive
+* @package fm-overlays
+* @description Front-end JS for fm-overlays.
+*
+*/
+
+/**
+ * Very simple overlay event handler.
+ *
+ * @returns {fmOverlay}
+ */
+function fmOverlay() {
+  const overlay = $('#fm-overlay');
+  const overlayWrapper = overlay.children('.fm-overlay-wrapper');
+  const timer = 500; // matches css transition duration
+  const activeClass = 'visible';
+  const closeButton = overlayWrapper.children('button.fm-overlay-close');
+
+  /**
+   * Hide overlay after fading out
+   */
+  function hideOverlay() {
+    overlay.removeClass(activeClass);
+
+    setTimeout(() => overlay.hide(), timer);
+  }
+
+  if (overlay.length) {
+    /**
+     * Display the overlay
+     */
+    setTimeout(() => overlay.show().addClass(activeClass), timer);
+
+    /**
+     * Exit strategies
+     * Escape (keyCode 27)
+     * Close Button
+     * Click Overlay
+     *
+     */
+    $(document).keyup((e) => {
+      if (e.keyCode === 27) {
+        hideOverlay();
+      }
+    });
+
+    closeButton.click(() => hideOverlay());
+
+    // close the overlay when a click occurs outside of the overlay content
+    overlay.children('.fm-overlay-fade').click(() => hideOverlay());
+  }
+}
+
+/**
+ * Initialize
+ */
+$(document).ready(() => fmOverlay());
```

Comment:
Throughout the entire JS files, don't we need to be adhering to the WP standards (e.g. spaces in parens, etc)?